src/server/system_services/bucket_server.js

```diff
@@ -1839,7 +1843,7 @@ function resolve_tiering_policy(req, policy_name) {
     return tiering_policy;
 }
 
-function can_delete_bucket(system, bucket) {
```

Comment:
Unused var.
Not strictly a part of the fix, but thought to fix as I came across it.

Reply:
I think we also exporting this function. If you already started with this cleanup can you remove the export as well?